From c4c666907715435a6d1d2e00378228cee1df3c9b Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Mon, 26 Jan 2015 16:18:10 +0800
Subject: [PATCH 1424/1543] MLK-10147 imx_v7_defconfig: remove unnessary
 config

commit 93e7f4b38fc38c613b5bad921273e4a361074a8c from
git://git.freescale.com/imx/linux-2.6-imx.git

Remove unnessary config since phylib can handle Atheros AT8031 PHYs.

Signed-off-by: Fugang Duan <B38611@freescale.com>
---
 arch/arm/configs/imx_v7_defconfig |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/arch/arm/configs/imx_v7_defconfig b/arch/arm/configs/imx_v7_defconfig
index 9d0c7fe..b63d59b 100644
--- a/arch/arm/configs/imx_v7_defconfig
+++ b/arch/arm/configs/imx_v7_defconfig
@@ -143,7 +143,6 @@ CONFIG_SMC91X=y
 CONFIG_SMC911X=y
 CONFIG_SMSC911X=y
 # CONFIG_NET_VENDOR_STMICRO is not set
-CONFIG_AT803X_PHY=y
 CONFIG_USB_PEGASUS=m
 CONFIG_USB_RTL8150=m
 CONFIG_USB_RTL8152=m
@@ -180,7 +179,6 @@ CONFIG_SERIAL_FSL_LPUART=y
 CONFIG_SERIAL_FSL_LPUART_CONSOLE=y
 CONFIG_FSL_OTP=y
 CONFIG_HW_RANDOM=y
-CONFIG_IMX_SEMA4=y
 # CONFIG_I2C_COMPAT is not set
 CONFIG_I2C_CHARDEV=y
 # CONFIG_I2C_HELPER_AUTO is not set
-- 
1.7.5.4

